### PR TITLE
Updates workflows to remove the CTAP 2.1 flag

### PR DESCRIPTION
**Pull request by kaczmarczyck**
Necessary to submit before #248 to not fail the old tests. Just removes the workflows for 2.1, so #248 should be submitted right after this one.

- [x] Tests pass

### CODE REVIEW

File: deploy.py [hunk 1]
```diff
@@ -947,7 +947,16 @@ def main(args):
       dest="application",
       action="store_const",
       const="store_latency",
-      help=("Compiles and installs the store_latency example."))
+      help=("Compiles and installs the store_latency example which print "
```
Comment (jmichelp) [inline deploy.py:950]: nit: prints

File: examples/erase_storage.rs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
```
Comment (jmichelp) [inline examples/erase_storage.rs:1]: Why is this file part of this PR?

**Comment by kaczmarczyck** [comment]
Aaah sorry old habit. This PR goes into stable, not develop. My bad. Closing and opening a new PR.

File: deploy.py (hunk 1 above)
Comment (kaczmarczyck) [reply deploy.py:950]: Will be fixed in a different PR :)

**Pull request closed without merging**